Shading control update

jmarrec opened the pull request:
A dummy PR to compare between my EnergyPlus-9 branch and the Shading Control specific changes, to discuss with @macumber

jmarrec committed:
Update ProposedE+.idd

jmarrec committed:
Update OpenStudio comments and options only, no new fields yet

jmarrec committed:
Correct small typo in E+ idd

jmarrec committed:
Crude way to update OpenStudio.idd. There's probably something smarter here, just for reference 

@macumber

macumber commented on openstudiocore/resources/energyplus/ProposedEnergy+.idd:
```diff
@@ -6932,18 +6922,30 @@ GlazedDoor:Interzone,
+       \required-field
+       \type object-list
+       \object-list ZoneNames
+  N1 , \field Shading Control Sequence Number
```
@jmarrec did Kyle miss this one?

macumber commented on openstudiocore/resources/model/OpenStudio.idd:
```diff
@@ -5345,7 +5342,19 @@ OS:ShadingControl,
        \note Referenced by surfaces that are exterior windows
        \note Not used by interzone windows
-  A3 , \field Shading Type
+  A3 , \field Zone Name
```
@jmarrec would really prefer this to be space level but that does cause issues at translation

macumber commented on openstudiocore/resources/model/OpenStudio.idd:
```diff
@@ -5291,13 +5291,7 @@ OS:SubSurface,
        \minimum 0
        \maximum 1
        \default autocalculate
-  A7, \field Shading Control Name
```
@jmarrec this seems like it would require some api changes.  My thought for right now is to preserve the current api and idd, just handle this at translation time.  So I think the changes to ProposedE+.idd are good, but we should not make changes to the OS IDD at this time.

jmarrec replied:
I agree. The commit with changes to OS Idd was marked as  "just for reference" for this reason

macumber committed:
Updates to IDD to diff more cleanly with release IDD

macumber committed:
Minor updates tp E+ IDD

macumber committed:
Revert IDD changes to OS IDD

macumber committed:
Update E+ SHA

macumber committed:
Update to shading controls

macumber committed:
Update to RunPeriod translation

macumber commented:
@jmarrec can you review my changes here?

jmarrec committed:
Merge remote-tracking branch 'origin/EnergyPlus-9-IddCleanup' into ShadingControlUpdate

jmarrec committed:
Update reference to field PV after renaming from dan

jmarrec committed:
Caught more fields that were renamed, should be it

jmarrec committed:
Catch syntax error in ProposedE+.idd

jmarrec committed:
Very Minor update

jmarrec committed:
ATU CV NoReheat adds an Inlet node before ATU which Uncontrolled didn't have, so update Gtest

jmarrec committed:
Remove one required-field where there is a default value to suppress IDD warning

jmarrec committed:
Geometry test uses an IDF RunPeriod object, need to update to match new IDD

jmarrec committed:
Fix test - all model + energyplus tests pass now

FenestrationSurface:Detailed in E+ doesn't have a Shading Control Field anymore

jmarrec committed:
Fix iconlibrary Gtest, don't look for Uncontrolled, it was removed

jmarrec committed:
Window doesn't have Shading Control anymore in E+ idd

jmarrec committed:
This will make Units.cpp fail because it can't recognize the units, while BasedOnField will be ignored

jmarrec commented:
`daylighting_shades.rb` fails:

```
   ** Severe  ** <root>[WindowShadingControl][Shading Control 2] - Missing required property 'zone_name'.
   **  Fatal  ** Errors occurred on processing input file. Preceding condition(s) cause termination.
```
